pkg/csi/snapshot_restore.go

```diff
@@ -135,12 +137,13 @@ type SnapshotRestoreStepper interface {
 }
 
 type snapshotRestoreSteps struct {
-	validateOps       ArgumentValidator
-	versionFetchOps   ApiVersionFetcher
-	createAppOps      ApplicationCreator
-	dataValidatorOps  DataValidator
-	snapshotCreateOps SnapshotCreator
-	cleanerOps        Cleaner
+	validateOps          ArgumentValidator
+	versionFetchOps      ApiVersionFetcher
+	createAppOps         ApplicationCreator
+	dataValidatorOps     DataValidator
+	snapshotCreateOps    SnapshotCreator
+	cleanerOps           Cleaner
+	SnapshotGroupVersion *metav1.GroupVersionForDiscovery
```

Comment:
Does it need to be exported?
```suggestion
	snapshotGroupVersion *metav1.GroupVersionForDiscovery
```